src/components/app/ListOfPublishedProfiles.css

```diff
@@ -58,6 +58,14 @@
   background: var(--grey-90-a05);
 }
 
+/**
+ * Overcome the selector specificity of the focus by including the "a"
+ */
```

Comment:
I'm surprised that this is needed, because a class name will always have a higher specificity than a tag name. Can you please double-check than this is needed?